exercicios-para-casa/exercicios-taline/uriJudge/04-exercicio/scrip.js

```diff
@@ -0,0 +1,20 @@
+let x = 0
```

Comment:
Talite fiz alguns ajustes do seu exercicio

let x = 0;
let alcool = 0;
let gasolina = 0;
let diesel = 0;
Como você declarou uma variavel x = 0, você não precisa declarar ela novamente dentro do while, você atribui um novo valor a ela, a condição eu mudei enquanto x for diferente de 4, ele vai entrar no loop, e nos cases do switch não a necessidade de usar o operador de == comparação, por debaixo dos panos proprio javascript já faz isso.

while (x !== 4) {
  x = Number(prompt("Insira o tipo de combustivel que voce prefere"));
  switch (x) {
    case 1:
      alcool += 1;
      break;
    case 2:
      gasolina += 1;
      break;
    case 3:
      diesel += 1;
      break;
      defaut: console.log("MUITO OBRIGADO", alcool, gasolina, diesel);
  }
}